pythran/tables.py

```diff
@@ -626,7 +626,8 @@ def update_effects(self, node):
         "std_": ConstMethodIntr(args=('a', 'axis', 'dtype'),
                                 defaults=(None, None)),
         "subtract": ConstFunctionIntr(),
-        "sum": ConstMethodIntr(),
+        "sum": ConstMethodIntr(argument_effects=(ReadOnceEffect(),
```

Comment:
Because of the array value and axis value (the only arguments we support for sum)